proj/src/main/webapp/static/chat.html

```diff
@@ -1,44 +1,51 @@
 <!DOCTYPE html>
-<html lang="en" dir="ltr">
-  <head>
-    <meta charset="utf-8">
-    <title>Chat Room</title>
+<html >
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="generator" content="Codeply" />
+    <title>Chat</title>
+    <base target="_self"> 
 
-    <!-- Bootstrap Scripts -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
-    
-
-    <!-- Firebase Scripts -->
-    <!-- The core Firebase JS SDK is always required and must be listed first -->
+    <!-- firebase -->
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-app.js"></script>
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-analytics.js"></script>
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-auth.js"></script>
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-database.js"></script>
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-messaging.js"></script>
     <script src="../app.js"></script>
 
-    <!-- Font  -->
-    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="../style.css">
-    </head>
-    
-    <body onload="init()">
-        <div class="container screen chat-page full-height">
-            <div class="row">
 
-            <div class="col sidebar">
+    <!-- font awesome icons -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <script src="https://kit.fontawesome.com/d691fbec03.js" crossorigin="anonymous"></script>
+
+    <!-- bootstrap -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" />
+
+    <!-- chatroom styling -->
+    <link rel="stylesheet" href="chatstyle.css" />
+</head>
 
-                <aside id="user-profile"  style="height:0">
-                    <p id="user-display-name"></p>
-                    <img id="user-pfp">
-                    <pre id="user-bio"></pre>
-                    <ul id="user-tags"></ul>
-                    <button type="button" id="close-settings">close</button>
-                </aside>
+<body onload="init()" id="top">
 
-                <aside id="sidebar">
+    <div class="row" id="body-row">
+        <!-- Sidebar -->
+        <div id="sidebar-container" class="sidebar sidebar-expanded d-none d-md-block">
+            <!-- d-* hiddens the Sidebar in smaller devices. Its itens can be kept on the Navbar 'Menu' -->
+            <!-- start of scroll -->
+            <!--<div class="scroll" id="profile-info>-->
```

Comment:
Should this not be used? If so, can we remove it rather than commenting out the div.

Reply:
There was an issue when the screen height is too small, the overflow is hidden. When I made it a scroll, it seemed to be scrolling the whole page rather than scrolling the div.  I was thinking that the scroll was not working because the div was cutting between list items

Reply:
I am currently working on this, it's a bit buggy because of the anchored collapse navigation